test/tune/test_flaml_raytune_consistency.py

```diff
@@ -0,0 +1,96 @@
+# import unittest
+import numpy as np
+# require: pip install flaml[blendsearch, ray]
+# require: pip install flaml[ray]
+import time
+from flaml import tune
+from ray import tune as raytune
+
+
+def evaluate_config(config):
+    '''evaluate a hyperparameter configuration'''
+    # we uss a toy example with 2 hyperparameters
+    metric = (round(config['x']) - 85000) ** 2 - config['x'] / config['y']
+    # usually the evaluation takes an non-neglible cost
+    # and the cost could be related to certain hyperparameters
+    # in this example, we assume it's proportional to x
+    time.sleep(config['x'] / 100000)
+    # use tune.report to report the metric to optimize
+    tune.report(metric=metric)
+
+
+config_search_space = {
+    'x': tune.lograndint(lower=1, upper=100000),
+    'y': tune.randint(lower=1, upper=100000)
+}
+
+low_cost_partial_config = {'x': 1}
+
+
+def setup_searcher(searcher_name):
+    from flaml.searcher.blendsearch import BlendSearch, CFO, RandomSearch
+    if 'cfo' in searcher_name:
+        searcher = CFO(space=config_search_space, low_cost_partial_config=low_cost_partial_config)
+    elif searcher_name == 'bs':
+        searcher = BlendSearch(
+            metric="metric", mode="min",
+            space=config_search_space,
+            low_cost_partial_config=low_cost_partial_config)
+    elif searcher_name == 'random':
+        searcher = RandomSearch(space=config_search_space)
+    else:
+        return None
+    return searcher
+
+
+def test_flaml_raytune_consistency(num_samples=-1, max_concurrent_trials=1, searcher_name='cfo'):
```

Comment:
when we perform automatic testing, this function will be called.
```suggestion
def _test_flaml_raytune_consistency(num_samples=-1, max_concurrent_trials=1, searcher_name='cfo'):
```